src/client/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
-add_executable(client main.cpp)
+add_library(helper)
+add_executable(client main.cpp helper/parser.h)
 target_link_libraries(client PRIVATE Crow::Crow)
```

Comment:
So cmake needs some slightly different syntax (add_executable is for binaries, not libraries, it should also be cleaner to just build the parser as a part of the client target). If you can set up cmake on your machine you should be able to use it with MSVC, otherwise I can fix it later. Let's talk about it at the meeting